### PR TITLE
Fix analyzer warnings

### PR DESCRIPTION
**Pull request by danielkbx**
## Description

Fixes analyzer warnings (which should have been fixed years ago).

### CODE REVIEW

**Comment by beatrupp** [comment]
The maintainership takeover by Microsoft seems to work pretty well. Not.

**Comment by DmitriyKirakosyan** [comment]
Hi @beatrupp , thank you for bumping this thread. I'll check why CI is failing and test the changes.

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,7 @@ xcuserdata/
 
 # Mac
 .DS_Store
+
+# Swift Package Manager
+.swiftpm
```
Comment (DmitriyKirakosyan) [inline .gitignore:37]: What are these changes for?

File: Source/PLCrashSysctl.c [hunk 2]
```diff
@@ -172,7 +172,6 @@ size_t plcrash_sysctl_valid_utf8_bytes_max (const uint8_t *s, size_t maxlen) {
         size_t seqlen = 0;
         if ((c & 0x80) == 0) {
             /* 1 byte sequence. Code point value range is 0 to 127. */
-            seqlen = 0;
```
Comment (MatkovIvan) [inline Source/PLCrashSysctl.c:175]: Can you comment it out instead of removing to keep the things more explicit?

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request closed without merging**